terminus-ui/src/select/select.component.ts

```diff
@@ -954,6 +980,15 @@ export class TsSelectComponent implements
         this.autocompleteFormControl.setValue(this.ngControl.value);
         this.autocompleteSelections = this.ngControl.value;
       }
+      if (this.ngControl.valueChanges) {
+        this.ngControl.valueChanges
+          .pipe(untilComponentDestroyed(this))
+          .subscribe((newValue) => {
+            if (this.autocomplete && newValue) {
```

Comment:
we can add `// istanbul ignore else` above this if so coverage will ignore the non-existent 'else' path